### DISCUSSION TITLE
admin - vendor - clint

### DISCUSSION THREAD

**Discussion by odaymusic** [RobertCraigie/prisma-client-py#650] [Q&A]
how to make 3 permissions and rules with prisma and flask ? 
which the best Prisma with node and express or prisma with flask ?

**Comment by RobertCraigie** [comment]
I'm not sure exactly what you mean by permissions and rules? Could you expand on that?

> which the best Prisma with node and express or prisma with flask ?

It's personal preference. I think most people would say the Node client as it is more fleshed out but if you don't like writing TypeScript you're not going to like it.
